crowbar_framework/config/routes.rb

```diff
@@ -240,4 +236,48 @@
   # TODO(must): Get rid of this wildcard route
   match "/:controller/:action/*(:.format)",
     via: [:get, :post, :put, :patch, :delete]
+
+  namespace :api,
```

Comment:
one problem that we have is the `backups_controller`.
the old one handles UI requests, and the new one json API requests. and if we want to use both, we somehow have to namespace them.

but i think overall you see this `/api` namespacing a lot when you google for it.

not sure how difficult would be to add an `api.crowbar.com` subdomain? that would be nice too...